.changes/0d2cc7ea-39dc-11ed-a261-0242ac120002.json

```diff
@@ -0,0 +1,6 @@
+{
+  "id": "0d2cc7ea-39dc-11ed-a261-0242ac120002",
+  "type": "feature",
+  "description": "Enables configurability of the retry strategy through environment variables, system properties, and AWS profiles.",
```

Comment:
**Nit**: Our changelog entries use the imperative present tense for the leading verb (i.e., "Enable" rather than "Enables"). This helps ensure a fluidity and consistency for the generated **CHANGELOG.md**.

This is documented in our [Git commit guidelines](https://github.com/awslabs/smithy-kotlin/blob/main/CONTRIBUTING.md#git-commit-guidelines) but not our [Changelog guidelines](https://github.com/awslabs/smithy-kotlin/blob/main/CONTRIBUTING.md#changelog), which is probably an oversight.

Reply:
I missed this, thanks! Updated and shipped.